app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsAdapter.kt

```diff
@@ -69,7 +71,31 @@ class ChatRoomsAdapter(private val context: Context,
         }
 
         private fun bindName(chatRoom: ChatRoom, textView: TextView) {
-            textView.content = chatRoom.name
+            textView.textContent = chatRoom.name
+
+            var drawable: Drawable? = null
+            when (chatRoom.type) {
+                is RoomType.Channel -> {
+                    drawable = DrawableHelper.getDrawableFromId(R.drawable.ic_room_channel, context)
+                }
+                is RoomType.PrivateGroup -> {
+                    drawable = DrawableHelper.getDrawableFromId(R.drawable.ic_room_lock, context)
+                }
+                is RoomType.DirectMessage -> {
```

Comment:
What if the server settings uses the real name instead of username? We should show the `ic_room_dm` icon only when showing usernames.